websites/migrations/0062_remove_course_introduction_lrt.py

```diff
@@ -0,0 +1,92 @@
+from collections import defaultdict
+
+from django.db import migrations, transaction
+from django.db.models import Q
+
+CONTENT_TYPE_METADATA = "sitemetadata"
+CONTENT_TYPE_WEBSITE = "website"
+COURSE_INTRODUCTION_LRT = "Course Introduction"
+
+
+def remove_course_intro_lrt(apps, schema_editor):
+    Website = apps.get_model("websites", "Website")
+    WebsiteContent = apps.get_model("websites", "WebsiteContent")
+    with transaction.atomic():
+        websites_to_update = Website.objects.filter(
+            metadata__learning_resource_types__contains=COURSE_INTRODUCTION_LRT
+        )
+        contents_to_update = WebsiteContent.objects.filter(
+            Q(type=CONTENT_TYPE_METADATA) | Q(type=CONTENT_TYPE_WEBSITE),
+            metadata__learning_resource_types__contains=COURSE_INTRODUCTION_LRT,
+        )
+
+        remove_course_intro_from_lrt(websites_to_update)
+        remove_course_intro_from_lrt(contents_to_update)
+
+
+def remove_course_intro_from_lrt(items):
+    for item in items.iterator():
+        metadata = item.metadata or {}
+        metadata["learning_resource_types"].remove(COURSE_INTRODUCTION_LRT)
+        item.metadata = metadata
+        item.save(update_fields=["metadata"])
+
+
+def restore_course_intro_lrt(apps, schema_editor):
+    Website = apps.get_model("websites", "Website")
+    WebsiteContent = apps.get_model("websites", "WebsiteContent")
+    with transaction.atomic():
+        websites_to_update = Website.objects.exclude(
+            metadata__learning_resource_types__contains=COURSE_INTRODUCTION_LRT
+        )
+        contents = (
+            WebsiteContent.objects.filter(
+                website_id__in=[w.pk for w in websites_to_update]
```

Comment:
nitpick - this works the same
```suggestion
                website__in=websites_to_update
```

Reply:
Updated in cc6395e3cf1195aee60ca9f4f07b6fb9d3448ece.